src/libcollections/binary_heap.rs

```diff
@@ -228,6 +228,15 @@ pub struct PeekMut<'a, T: 'a + Ord> {
     sift: bool,
 }
 
+#[stable(feature = "collection_debug", since = "1.15.0")]
+impl<'a, T: Ord + fmt::Debug> fmt::Debug for PeekMut<'a, T> {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        f.debug_tuple("PeekMut")
```

Comment:
To match the others, should this not be `"BinaryHeap::PeekMut"`? Although it might be better to use part of the actual path like `"binary_heap::PeekMut"` for all of these instead.